example/403.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Access Denied</title>
+    <style>
+      body {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 100vh;
+        margin: 0;
+        font-family: 'Arial', sans-serif;
+        background-color: #328ba8;
+        color: #ffffff;
+      }
+
+      .access-denied-container {
+        text-align: center;
+        padding: 50px;
+        border-radius: 10px;
+        background: rgba(0, 0, 0, 0.7);
+      }
+
+      h1 {
+        font-size: 5em;
+        margin: 0;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="access-denied-container">
+      <h1>Access Denied</h1>
+      <p>Host: {{.Host}}</p>
```

Comment:
maybe its a dumb question, but no chances of XSS here?

Reply:
That's a good point. This is just to demonstrate the templating capability. How would you suggest demonstrating the templating feature?

Reply:
Maybe use `stripHTML`?
```
stripHTML

Removes HTML from a string.

{{stripHTML "Shows <b>only</b> text content"}}
```

Reply:
Done.